src/legacy/core_plugins/kbn_vislib_vis_types/public/vislib/_variables.scss

```diff
@@ -1,4 +1,5 @@
 // TODO: Use the same styles for TSVB and Vislib vis'
+// Duplicate of src/legacy/ui/public/vis/vis_types/vislib_vis_legend/_variables.scss
```

Comment:
Instead of duplicating this file which will then cause them to be out of sync, in the `_index.scss` file, you can just import this path mentioned in the comment.

Reply:
I still see that these files are being duplicated. Let me know if you want me to push a fix for it.